WordPressAuthenticatorTests/Authenticator/WordPressAuthenticatorTests.swift

```diff
@@ -1,10 +1,20 @@
 import XCTest
 @testable import WordPressAuthenticator
 
-
 // MARK: - WordPressAuthenticator Unit Tests
 //
 class WordPressAuthenticatorTests: XCTestCase {
+    let timeInterval = TimeInterval(3)
```

Comment:
What do you think of this variable name to explain to the reader that this value is the default timeout to use in the async expectations?

```suggestion
    let timeout = TimeInterval(3)
```